docs/jitserver.md

```diff
@@ -26,14 +26,14 @@
 
 ![Start of content that applies to Java 8 and later](cr/java8.png)
 ![Start of content that applies to Java 11 and later](cr/java11.png)  
-**Linux&reg; on x86-64 only**
+**Linux&reg; on x86, IBM Power&reg; systems, and IBM Z&reg; systems (64-bit only)**
```

Comment:
I would like to be more specific here because we only support Linux platforms, thus AIX and zOS are not supported, so maybe:
`Linux&reg; on x86, Linux on Power&reg; systems, and Linux on IBM Z&reg; systems (64-bit only)`

Reply:
Fixed.